systemtest/src/test/java/io/strimzi/systemtest/MultipleNamespaceST.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+import io.fabric8.kubernetes.api.model.Job;
+import io.strimzi.test.ClusterOperator;
+import io.strimzi.test.Namespace;
+import io.strimzi.test.StrimziExtension;
+import io.strimzi.test.TestUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.io.File;
+import java.util.List;
+
+import static io.strimzi.test.StrimziExtension.REGRESSION;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.not;
+
+@ExtendWith(StrimziExtension.class)
+@Namespace(MultipleNamespaceST.DEFAULT_NAMESPACE)
+@Namespace(value = MultipleNamespaceST.SECOND_NAMESPACE, use = false)
+@ClusterOperator
+class MultipleNamespaceST extends AbstractST {
+    private static final Logger LOGGER = LogManager.getLogger(KafkaST.class);
+    static final String DEFAULT_NAMESPACE = "multiple-namespace-test";
+    static final String SECOND_NAMESPACE = "topic-operator-namespace";
+    private static final String TOPIC = "my-topic";
+    private static final String TOPIC_INSTALL_DIR = "../examples/topic/kafka-topic.yaml";
+    private static final int MESSAGE_COUNT = 20;
+
+    private static Resources classResources;
+    private static Resources secondNamespaceResources;
+
+
+    /**
+     * Test the case where the TO is configured to watch a different namespace that it is deployed in
+     */
+    @Test
+    @Tag(REGRESSION)
+    void testWatchingOtherNamespace() throws InterruptedException {
```

Comment:
Method name isn't clear about what might be watching multiple NS → `testTopicOperatorWatchingOtherNamespace()`

Reply:
Test name changed